src/FurCoNZ.Web/Views/Home/News.cshtml

```diff
@@ -0,0 +1,18 @@
+﻿
+@{
+    ViewData["Title"] = "News";
+}
+
+<h1>News</h1>
+
+@* TODO: Generate this content from the database *@
```

Comment:
I think at this stage it might be easiest to handle this manually, but certainly good for 2021.

Reply:
Added #83 to track this.